docs/how_to/implement_the_right_toggle_type.rst

```diff
@@ -51,16 +51,22 @@ For information on `Configuration Models`_ refer to that library.
 
 .. _Configuration Models: https://github.com/openedx/django-config-models/
 
-Creating toggles
------------------
+Creating and naming toggles
+----------------------------
 
 Flags and switches are created with a name. The namespace prefix is used to categorize them and prevent conflicts.
 
 .. code:: python
 
-    FLAG_FOO = WaffleFlag('namespace.feature', module_name=__name__)
-    SWITCH_BAR = WaffleSwitch('namespace.feature', module_name=__name__)
-    SETTING_TOGGLE_BAZ = SettingToggle("SETTING_NAME", default=False, module_name=__name__)
+    FOO_FLAG = WaffleFlag('namespace.some_feature', module_name=__name__)
+    BAR_SWITCH = WaffleSwitch('namespace.some_feature', module_name=__name__)
+    BAZ_TOGGLE = SettingToggle("SOME_SETTING_TOGGLE", default=False, module_name=__name__)
+
+When naming a setting to be used as a toggle, consider the following recommendations:
+
+- Use the suffix ``_TOGGLE`` to clarify that the setting is being used as a toggle.
+- Avoid ``ENABLE`` or ``ENABLED`` in the name. It is redundant when accessing toggles.
+- Avoid ``DISABLE`` or ``DISABLED`` in the name. It makes accessing the toggle very confusing (e.g. ``DISABLE_MY_FEATURE.is_enabled()``). Consider defaulting the toggle to True instead.
```

Comment:
Non-blocking: This is good guidance to have for switches and flags, too. Here's a suggestion of how you could generalize the guidance to all three toggle types:

```suggestion
When naming a toggle, consider the following recommendations:

- Use the suffixes ``_FLAG`` or ``_SWITCH`` to indicate Waffle flags and switches, respectively. Use the suffix ``_TOGGLE`` to clarify that a setting is being used as a toggle.
- Avoid ``ENABLE`` or ``ENABLED`` in the name. It is redundant when accessing toggles (e.g. ``ENABLE_MY_EXPERIMENT_FLAG.is_enabled()``).
- Avoid ``DISABLE`` or ``DISABLED`` in the name. It makes accessing the toggle very confusing (e.g. ``DISABLE_MY_FEATURE.is_enabled()``). For settings toggles, consider defaulting the toggle to True instead. For Waffle switches and flags, which must default to False, consider re-framing them to positively describe the enabled state (e.g. ``LEGACY_EXPERIENCE_FLAG`` instead of ``DISABLE_NEW_EXPERIENCE_FLAG``).
```